front/src/utils/share/share.util.test.ts

```diff
@@ -0,0 +1,42 @@
+import { ShareUtils } from "..";
+
+describe("Share utils", () => {
+  const title = "Test title";
+  const message = "Test message";
+  const url = "https://testurl.com";
+
+  describe("buildShareContent", () => {
+    it("buildShareContent is called with url on Android", () => {
+      const expectdMessage = `${message} ${url}`;
+      const expectedResult = { message: expectdMessage, title, url };
+      const isAndroid = true;
+      expect(
+        ShareUtils.buildShareContent(isAndroid, title, message, url)
+      ).toEqual(expectedResult);
+    });
+
+    it("buildShareContent is called without url on Android", () => {
```

Comment:
`should return share content with simple message when url is undefined on Android`

mais je me demande si c'est vraiment un comportement souhaitable ?